testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/common/kafka/log4j-kafka.properties.template

```diff
@@ -0,0 +1,28 @@
+# https://github.com/confluentinc/kafka-images/blob/master/kafka/include/etc/confluent/docker/log4j.properties.template
+
+log4j.rootLogger={{ env["KAFKA_LOG4J_ROOT_LOGLEVEL"] | default('WARN') }}, stdout
+
+log4j.appender.stdout=org.apache.log4j.ConsoleAppender
+log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
+log4j.appender.stdout.layout.ConversionPattern=[%d] %p %m (%c)%n
+
+{% set loggers = {
+  'kafka': 'WARN',
+  'kafka.network.RequestChannel$': 'WARN',
+  'kafka.producer.async.DefaultEventHandler': 'WARN',
+  'kafka.request.logger': 'WARN',
+  'kafka.controller': 'WARN',
+  'kafka.log.LogCleaner': 'WARN',
+  'state.change.logger': 'WARN',
+  'kafka.authorizer.logger': 'WARN'
+  } -%}
+
+{% if env['KAFKA_LOG4J_LOGGERS'] %}
+{% set loggers = parse_log4j_loggers(env['KAFKA_LOG4J_LOGGERS'], loggers) %}
+{% endif %}
+
+{% for logger,loglevel in loggers.items() %}
+log4j.logger.{{logger}}={{loglevel}}
+{% endfor %}
+
+log4j.logger.kafka.server.KafkaServer=INFO
```

Comment:
Turns out the environment depends on a `started (kafka.server.KafkaServer)` log message.
https://github.com/trinodb/trino/blob/756be8799efea3f8d06bc63f41d694909dc829e9/testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/common/Kafka.java#L97